### PR TITLE
typo in one line with student id and positive word

### PR DESCRIPTION
**Pull request by jonduan**
After the correction, the number in the next paragraph matches the number generated by the R code chunk.

### CODE REVIEW

**Comment by jonduan** [comment]
delete a duplicated paragraph and correct two typoes.

**Comment by jrosen48** [comment]
Thanks much for all of these @jonduan, will review.

File: 06-wt-multilevel-models-1.Rmd [hunk 1]
```diff
@@ -78,7 +78,7 @@ datasets. We assign each of the datasets a name using `<-`.
 ``` {r}
 # Gradebook and log-trace data for F15 and S16 semesters
 s12_course_data <-
-  read_csv(here(
+  read_csv(here::here(
```
Comment (jrosen48) [inline 06-wt-multilevel-models-1.Rmd]: thanks for adding this - do you think this is necessary even though we have the here package loaded?

File: 06-wt-multilevel-models-1.Rmd [hunk 2]
```diff
@@ -466,13 +466,13 @@ with `sjPlot::tab_model()`, as well as a file name that ends in `.doc` to the
...
 ```{r}
-apaTables::apa.reg.table(m_linear)
+(m_summary = apaTables::apa.reg.table(m_linear))
 ```
 
 Helpfully, you can save the output to a Word document, by adding a `filename` argument:
 
 ```{r, eval = FALSE}
-apaTables::apa.reg.table(m_summary, filename = "regression-table-output.doc")
+apaTables::apa.reg.table(m_linear, filename = "regression-table-output.doc")
```
Comment (jrosen48) [inline 06-wt-multilevel-models-1.Rmd]: good catch! thanks

File: 06-wt-multilevel-models-1.Rmd [hunk 3]
```diff
@@ -466,13 +466,13 @@ with `sjPlot::tab_model()`, as well as a file name that ends in `.doc` to the
 `filename` argument, i.e.:
 
 ```{r}
-apaTables::apa.reg.table(m_linear)
+(m_summary = apaTables::apa.reg.table(m_linear))
```
Comment (jrosen48) [inline 06-wt-multilevel-models-1.Rmd]: Thanks. Wondering if we should actually delete this entire chunk (as it's the next chunk which specifies a filename, as the above code suggests)? Or, do you think it's worthwhile to keep this, to show how this output can be saved to an object?

File: 07-wt-multilevel-models-2.Rmd [hunk 4]
```diff
@@ -92,7 +92,7 @@ Let's look at the results.
 
 ```{r, eval = FALSE}
 iris_with_dummy_codes %>% 
-    count(Species, Speciessetosa, Speciesversicolor, Speciesvirginica)
+    dplyr::count(Species, Speciessetosa, Speciesversicolor, Speciesvirginica)
```
Comment (jrosen48) [inline 07-wt-multilevel-models-2.Rmd]: like above - given that dplyr is loaded, do you think this must be included?

File: 07-wt-multilevel-models-2.Rmd [hunk 5]
```diff
@@ -166,7 +166,7 @@ makes it easy to do this. This function allows us to re-order the levels within
 ``` {r}
 dat <-
   dat %>%
-  mutate(course_id = fct_relevel(course_id, "course_idPhysA-S116-01"))
+  mutate(course_id = fct_relevel(course_id, "PhysA-S116-01"))
```
Comment (jrosen48) [inline 07-wt-multilevel-models-2.Rmd]: did the former not work (with `course_id` prepended)? wondering if that was an artifact of an older dataset.

File: 07-wt-multilevel-models-2.Rmd [hunk 6]
```diff
@@ -228,7 +228,7 @@ identically distributed" (*i.i.d.*) residuals (Field, Miles, & Field, 2012).
 Multi-level models are a way to deal with the difficulty of interpreting the
 estimated effects for each of many groups, like classes, and to address the
 assumption of independence. Multi-level models do this by (still) estimating the
-effect of being a student in each group, but with a key distinctionfrom linear
+effect of being a student in each group, but with a key distinction from linear
```
Comment (jrosen48) [inline 07-wt-multilevel-models-2.Rmd]: thx

File: 07-wt-multilevel-models-2.Rmd [hunk 7]
```diff
@@ -334,7 +334,7 @@ which the classes are nested within the schools automatically (Bates, Maechler,
 Bolker, & Walker, 2015).
 
 ``` {r, eval = FALSE}
-# this model would specify a group effect for both the course nad school
+# this model would specify a group effect for both the course and school
```
Comment (jrosen48) [inline 07-wt-multilevel-models-2.Rmd]: thx

File: 06-wt-multilevel-models-1.Rmd (hunk 1 above)
Comment (restrellado) [reply 06-wt-multilevel-models-1.Rmd]: It's caused bugs for me in the past so I've been using `here::here()`. I can't remember which package it conflicts with though.

**Comment by restrellado** [comment]
Thank you so much for submitting this! I'll work on the text analysis one. I want to double check because I know we swapped in `distinct` for `unique` and I'm wondering if the change has anything to do with that. @joshuarosenberg can we leave this open for a bit?

**Comment by jrosen48** [comment]
Yes of course! Sounds good. Thanks again @jonduan, on behalf of all of us.

**Comment by jrosen48** [comment]
@restrellado do you think we can merge the changes not in the PR you mentioned above?

**Comment by restrellado** [comment]
> @restrellado do you think we can merge the changes not in the PR you mentioned above?

Yeah I think so but it might have to be in a separate PR because I'm seeing some conflicts. We could resolve in the command line but this late in the game might be better to play it safe and fix it in a separate PR with kudos to @jonduan When you're done with that you can close this PR

**Comment by joshuarosenberg** [comment]
Makes sense. I can go ahead and do this (with kudos intact)!

File: 06-wt-multilevel-models-1.Rmd (hunk 1 above)
Comment (jrosen48) [reply 06-wt-multilevel-models-1.Rmd]: kk, keeping here::here()

File: 07-wt-multilevel-models-2.Rmd (hunk 5 above)
Comment (jrosen48) [reply 07-wt-multilevel-models-2.Rmd]: confirming you were right here - thanks

File: 06-wt-multilevel-models-1.Rmd (hunk 1 above)
Comment (jrosen48) [reply 06-wt-multilevel-models-1.Rmd]: ok, this was a case for which we ended up loading data from the package, so is simplified/a bit clearer now (just `course_data <- dataedu::course_data)`).

**Comment by jrosen48** [comment]
confirming this is addressed in #230 and #219. Thank you so much @jonduan, will acknowledge you (on that note, tagging in #163).

**Pull request closed without merging**